src/jvmTest/resources/DESCRIPTION.md

```diff
@@ -0,0 +1,15 @@
+## Test description
```

Comment:
Description of tests is still incomplete and too scarce. Do not forget to improve it each time you add a benchmark!